impl-endpoints/cloudflare-go/runner.go

```diff
@@ -85,55 +88,92 @@ func (t testCaseDC) ServerConfig() (*tls.Config, error) {
 	return config, nil
 }
 
-func (t testCaseDC) ClientConnectionHandler(conn *tls.Conn) error {
+func (t testCaseDC) ClientConnectionHandler(conn *tls.Conn, err error) error {
+	if err != nil {
+		return fmt.Errorf("Unexpected error: \"%s\"")
+	}
 	if conn.ConnectionState().VerifiedDC == nil {
 		return errors.New("Failed to verify a delegated credential")
 	}
 	return nil
 }
 
-func (t testCaseDC) ServerConnectionHandler(conn *tls.Conn) error {
-	// TODO
-	return errors.New("NOT IMPLEMENTED")
+func (t testCaseDC) ServerConnectionHandler(conn *tls.Conn, err error) error {
+	panic("test case not implemented")
```

Comment:
Wouldn't it be better to return 64 as opposed to panicking?

Reply:
There's currently no convention. I like panicking here because it's more explicit that this feature is missing.

Will the convention be that unimplemented tests exit with code 64?

Reply:
The convention is to return the special signal for unsupported test cases. So we shouldn't panic here.

Reply:
Reverted. Though returning error doesn't actually result in exiting with error code 64. I'll fix this in a follow-up PR.

Reply:
See #39